pkg/driverbuilder/builder/ubuntu.go

```diff
@@ -16,98 +14,80 @@ import (
 //go:embed templates/ubuntu.sh
 var ubuntuTemplate string
 
-// TargetTypeUbuntuGeneric identifies the UbuntuGeneric target.
-const TargetTypeUbuntuGeneric Type = "ubuntu-generic"
+// TargetTypeUbuntu identifies the Ubuntu target.
+const TargetTypeUbuntu Type = "ubuntu"
 
-// TargetTypeUbuntuAWS identifies the UbuntuAWS target.
+// backwards compatibility
+const TargetTypeUbuntuGeneric Type = "ubuntu-generic"
```

Comment:
If you are willing to open 2 more PRs:
* one in kernel-crawler to set `ubuntu` as target for ubuntu configs
* one in test-infra to replace all `ubuntu-{generic,aws}` with just `ubuntu` target

i think we can go as far as fully dropping ubuntu-generic and ubuntu-aws. What do you think?
We can:
* drop ubuntu-generic and ubuntu-aws support here
* open the 2 PRs and leave them open
* as soon as the new driverkit is released, we bump the driverkit version inside the test-infra PR
* then we can merge both the kernel-crawler and the test-infra one

WDYT?

Reply:
IMHO we can also keep backward compatibility for now, and break it later (like for driverkit 1.0.0); but the kernel-crawler patch would still be needed.

Reply:
Yea, I agree with you, and plan to open those PRs if/when this goes through 😄 

I would keep them for now, personally, for the sake of backwards compatibility like you said. But since Driverkit is not even to a 1.0 release, it's probably fine to quickly deprecate.